src/dialogs/more-info/controls/more-info-sun.js

```diff
@@ -63,7 +63,11 @@ class MoreInfoSun extends LocalizeMixin(PolymerElement) {
   }
 
   itemCaption(type) {
-    return type === "ris" ? "Rising " : "Setting ";
+    if (type === "ris") {
+      return this.localize("ui.card.sun.rising");
+    } else {
```

Comment:
Remove the else. Just have the return under the if

Reply:
done :)